### PR TITLE
add multi checkbox input with chip display

### LINKED ISSUE #864: Multi-Checkbox Input
- The search should be optional
Image
Image

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #864

## [OPTIONAL] Give testing instructions to reviewers
Look at the notebook and also try the `enableSearch` and `useChipDisplay` options

### CODE REVIEW

File: lib/components/ChipList.tsx
Comment (MaxSchaefer) [inline lib/components/ChipList.tsx:1]: `Chip` and `ChipList` stories would be nice

**Comment by MaxSchaefer** [comment]
https://c3eb9c66.hw-web-storybook-lib.pages.dev/?path=/story/select--multi-select-variations&args=enableSearch:!true
@DasProffi I've enabled the search. I cannot find "Custom styled".

**Comment by DasProffi** [comment]
> https://c3eb9c66.hw-web-storybook-lib.pages.dev/?path=/story/select--multi-select-variations&args=enableSearch:!true
> @DasProffi I've enabled the search. I cannot find "Custom styled".
> 

The mapping of the label only works when the label is a string and not a React node, but that can be adapted

File: lib/components/user-input/MultiSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+  return (
+    <div className={tx(className)}>
+      {label && (
+        <label htmlFor={label} className={tx(' mb-1', labelClassName)}>
+          {label}
+        </label>
+      )}
+      <Menu<HTMLDivElement>
+        alignment="t_"
```
Comment (MaxSchaefer) [inline lib/components/user-input/MultiSelect.tsx:106]: Were is the closing generic?

**Review by MaxSchaefer — APPROVED** [review]

**Comment by MaxSchaefer** [comment]
Awesome

File: lib/components/user-input/MultiSelect.tsx (hunk 1 above)
Comment (DasProffi) [reply lib/components/user-input/MultiSelect.tsx:106]: You cannot close with a generic, you close with </Menu> at least according to my linter

**Pull request merged**